### PR TITLE
114 feature visualize target vs current speed

### LINKED ISSUE #114: [Feature]: Visualize target vs current speed
### Description

Add visualization of target vs current speed to test controller 

### Definition of Done

- Add useful visualization of target vs current speed to be able to test controllers

### PR DESCRIPTION
**Pull request by ghost**
# Description

These changes make it possible to view the current and the target speed of the ego vehicle in a diagram. This makes it possible to properly test and tune the PID velocity controller

Closes #114 

## Time invested

- Julian Graf: 2h

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## Does this PR introduce a breaking change?

No

## Most important changes

- Docs on how to use rqt_plot for visualizing the current and target speed
- the velocity controller node now republishes the current speed as a float value as rqt_plot cant read the velocity data provided by the rosbridge

I'm not sure if I put the documentation at the right place or if it would be better to give it another spot as it is not really part of the research anymore

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works (might be obsolete with CI later on)
- [x] New and existing unit tests pass locally with my changes (might be obsolete with CI later on)

### CODE REVIEW

File: doc/03_research/01_acting/03_acting_testing.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# How to test acting components
```
Comment (schwalga) [inline doc/03_research/01_acting/03_acting_testing.md:1]: As you already mentioned, I think it is time to open up a new folder for these kinds of documents.

**Review by schwalga — CHANGES_REQUESTED** [review]
Good work. Does exactly what it's supposed to do.
The only thing I would change is the location of the md-file. 
~10min @schwalga

File: doc/03_research/01_acting/03_acting_testing.md (hunk 1 above)
Comment (Julian-Graf) [reply doc/03_research/01_acting/03_acting_testing.md]: Fixed in 21267cb6dd92f8ee5dc715489ecf5cfd0f278ef0

**Review by schwalga — APPROVED** [review]
Looks good to me.

**Pull request merged**